.github/workflows/storybook.yml

```diff
@@ -162,6 +162,17 @@ jobs:
           cache: npm
       - name: npm install
         run: npm i -w assets -w storybook -w tests/js --include-workspace-root
-      # TODO: #10473 will complete the migration to Storybook test-runner as storyshots is deprecated.
-      # - name: Test Storybook
-      #   run: npm run test:storybook
+      - name: Install playwright browser
+        run: npx playwright install chromium
```

Comment:
Looking at the docs, there are a few considerations here.

1. There are also some amount of [system dependencies](https://playwright.dev/docs/browsers#chromium-headless-shell) to consider in addition to the browser itself. These can also be installed by `playwright` but is an opt-in part of the command
2. By default the full browser is installed, but it is possible to install a headless-only variant which would be more efficient in CI https://playwright.dev/docs/browsers#chromium-headless-shell